### PR TITLE
Add src9 component to account presets

### LINKED ISSUE #1200: Add SRC9 to Account presets.
This will make the transition to only accepting v3 transactions easier since accounts not having STRK to send transactions can use a paymaster to recover/transfer funds.

### PR DESCRIPTION
**Pull request by ericnordelo**
Fixes #1200  

#### PR Checklist

- [x] Tests
- [x] Documentation
- [x] Added entry to CHANGELOG.md 
- [ ] Tried the feature on a public network

### CODE REVIEW

File: docs/modules/ROOT/pages/api/account.adoc [hunk 1]
```diff
@@ -726,6 +727,7 @@ include::../utils/_class_hashes.adoc[]
 .AccountComponent
 
 * xref:#AccountComponent-Embeddable-Mixin-Impl[`++AccountMixinImpl++`]
+* xref:#SRC9Component-Embeddable-Impls-OutsideExecutionV2Impl[`++OutsideExecutionV2Impl++`]
```
Comment (andrew-fleming) [inline docs/modules/ROOT/pages/api/account.adoc:730]:
```suggestion

.SRC9Component

* xref:#SRC9Component-Embeddable-Impls-OutsideExecutionV2Impl[`++OutsideExecutionV2Impl++`]
```
We should have SRC9Component on the right side of `OutsideExecutionV2Impl`, no?

Comment (andrew-fleming) [reply docs/modules/ROOT/pages/api/account.adoc:730]: Same with eth account if we agree

**Review by andrew-fleming — APPROVED** [review]
I left a doc suggestion/question. Otherwise, LGTM!

File: docs/modules/ROOT/pages/api/account.adoc (hunk 1 above)
Comment (ericnordelo) [reply docs/modules/ROOT/pages/api/account.adoc:730]: Good catch!

**Pull request merged**